app/presenters/teams/filtered_list.rb

```diff
@@ -30,14 +30,18 @@ def team_id
 
     def filter_by_query
       teams.includes(:folders, folders: [:accounts]).where(
-        'lower(teams.name) LIKE :query
-        OR lower(folders.name) LIKE :query
+        'lower(accounts.description) LIKE :query
         OR lower(accounts.accountname) LIKE :query
-        OR lower(accounts.description) LIKE :query',
+        OR lower(folders.name) LIKE :query
+        OR lower(teams.name) LIKE :query',
         query: "%#{query}%"
       )
            .references(:folders,
                        folders: [:accounts])
+          .order("CASE WHEN lower(accounts.accountname) LIKE '%#{query}%' THEN 1 ELSE 2")
```

Comment:
What does this do? when the accountname matches order by 1 else by 2?

Reply:
Already removed.